pypose/module/dynamics.py

```diff
@@ -0,0 +1,196 @@
+        pass
+
+    def reset(self,t=0):
+        self.t.fill_(0)
+
+    @property
+    def A(self):
+        if hasattr(self, '_A'):
+            return self._A
+        else:
+            func = lambda x: self.state_trasition(x, self.input)
+            return jacobian(func, self.state, **self.jacargs)
+
+    @property
+    def B(self):
+        if hasattr(self, '_B'):
+            return self._B
+        else:
+            func = lambda x: self.state_trasition(self.state, x)
+            return jacobian(func, self.input, **self.jacargs)
+
+    @property
+    def C(self):
+        if hasattr(self, '_C'):
+            return self._C
+        else:
+            func = lambda x: self.observation(x, self.input)
+            return jacobian(func, self.state, **self.jacargs)
+ 
+    @property
+    def D(self):
+        if hasattr(self, '_D'):
+            return self._D
+        else:
+            func = lambda x: self.observation(self.state, x)
+            return jacobian(func, self.input, **self.jacargs)
+
+
+class LTI(_System):
+    r'''
+    A sub-class of '_System' to represent Linear Time-Invariant system.
+    
+    Args:
+        A, B, C, D (:obj:`Tensor`): The coefficient matrix in the state-space equation of LTI system
+        c1, c2 (:obj:`Tensor`): Bias generated by system
+        
+    Note:
+        According to the actual physical meaning, the dimensions of A, B, C, D must be the same,
+        whether in the batch case or not.
+        
+        The system is time invariant.
+    '''
+    def __init__(self, A, B, C, D, c1=None, c2=None):
+        super(LTI, self).__init__(time=False)
+        assert A.ndim == B.ndim == C.ndim == D.ndim, "Invalid System Matrices dimensions"
+        self._A, self._B, self._C, self._D = A, B, C, D
+        self._c1, self._c2 = c1, c2
+
+    @property
+    def c1(self):
+        return self._c1
+    
+    @property
+    def c2(self):
+        return self._c2
+    
+    def forward(self, x, u):
+        r'''
+        Args:
+            x (:obj:`Tensor`): The state of LTI system,
+            u (:obj:`Tensor`): The input of LTI system.
+
+        Return:
+            Tensor: The solved batched tensor, which means state-transition and observation of the system.
+
+            
+        Every linear time-invariant lumped system can be described by a set of equations of the form
+        which is called the state-space equation.
+        
+        .. math::
+            \begin{align*}
+                z_{i} = A_{i} \times x_{i} + B_{i} \times u_{i} + c_1 \\
+                y_{i} = C_{i} \times x_{i} + D_{i} \times u_{i} + c_2 \\
+            \end{align*}
+            
+        where :math:`x` is actually :math:`\dot{x}`, the differential form of :math:`x`.
+        
+        Let the input be matrix :math:`A`, :math:`B`, :math:`C`, :math:`D`, :math:`x`, :math:`u`, :math:`c_1`, :math:`c_2`. 
+        :math:`x_i` represents each individual matrix in the batch. 
+        
+        Note:
```

Comment:
I'm glad to see you put these details in this note. That means you are clear about them. Great! 
However, I think for this software doc, we don't have to provide these kind of details. Common users should be able to figure them out. 
I suggest we only keep the item " x, u, A, B, C, D can be single input or in a batch. In the batch case, their dimension must be consistent." You can polish this sentence. Make sure the symbols are in bold.